### ISSUE TITLE
rule: padded-blocks: false positive with comments

### ISSUE THREAD

**Issue by joeybaker** [babel/babel-eslint#33]
With `padded-blocks` set to `"never"` the following is reported as a warning and should not be:

```js
if (a){
  // i'm a comment!
 let b = c
}
```

Removing the comment allows the rule to pass. The problem does not exist with the default eslint parser.

**Comment by sebmck** [comment]
Comments aren't currently attached. Might be a bit tricky to get them in the same format as ESLint.

**Comment by brigand** [comment]
+1 I'm gradually going through a few hundred files and making them pass lint; but I have to leave it on and keep track of which lines were false positives.

**Comment by sebmck** [comment]
Should be fixed via #76 and releasde as of 3.1.0.

**Comment by DenisIzmaylov** [comment]
The same issue for decorators when I use babel-eslint parser:
```
test/components/connect.spec.js
  50:43  error  Block must not be padded by blank lines  padded-blocks
```
Where lines 50-52 contains:
```javascript
    it('sets displayName correctly', () => {
      @connect(state => state)
      class Container extends Component {
...
```

https://github.com/eslint/eslint/issues/1416

**Comment by hzoo** [comment]
Can you make a separate issue for this?
